packages/ui-rating/package.json

```diff
@@ -38,9 +38,10 @@
   },
   "devDependencies": {
     "@instructure/ui-babel-preset": "8.53.2",
-    "@instructure/ui-test-locator": "8.53.2",
-    "@instructure/ui-test-utils": "8.53.2",
-    "@instructure/ui-themes": "8.53.2"
+    "@instructure/ui-themes": "8.53.2",
+    "@instructure/ui-axe-check": "8.53.1",
```

Comment:
Is there a particular reasion why you use 8.53.1 here instead of the latest?

Reply:
Good catch thanks!
It was probably a bad merge conflict resolve.